src/HelloScala.scala

```diff
@@ -0,0 +1,48 @@
+
+//The three main components of a Macbook.
+// Each MacbookPro is made by Apple, has a certain screen size, measured in inches, and RAM capacity and
+//hard drive space, measured in gigabites.
+
+case class MacbookPro(model: String, size: Int, ram: Int, storage: Int)
+
+
+//Three components of an iMac. Includes screen size, ram size and storage capacity.
+case class iMac(model: String, size: Int, ram: Int, storage: String)
+
+
+//Three componenets of an iPadAir. Includes screen size, ram size and storage capacity.
+case class iPadAir(model: String, size: Double, ram: Int, storage: String)
+
+object HelloScala {
+
+  def main(args: Array[String]): Unit = {
+
+    val p = MacbookPro("MacbookPro", 13, 8, 128)
+    val proModel = p.model
+    var screenSize = p.size
+    var ramSize = p.ram
+    var hardDrive = p.storage
```

Comment:
Makine some fields var and some val does not give you practice on the concept unless you actually make changes to the var types, right?  Make sense?